src/stravalib/tests/integration/test_client.py

```diff
@@ -15,6 +15,14 @@
 from stravalib.unithelper import UnitConverter, meters, miles
 
 
+@pytest.fixture
```

Comment:
made this a fixture to clean up the test a bit. however i'm still unsure about the values in example_zone_response (why we have floats and why the test passed before). i'm certain i'm missing something!